lib/DxcSupport/HLSLOptions.cpp

```diff
@@ -499,6 +499,8 @@ int ReadDxcOpts(const OptTable *optionTable, unsigned flagsToInclude,
     opts.EnableOperatorOverloading = true;
     // Enable template support
     opts.EnableTemplates = true;
+    // Enable Unions support
+    opts.EnableUnions = true;
```

Comment:
This should be gated on HLSL 202x rather than 2021. You'll need to rebase your branch to get the new version support.

Reply:
Done